### PR TITLE
no-ops evaluator example

### PR DESCRIPTION
**Pull request by nankolena**
Example commit for a no-ops semantic segmentation evaluator. Will not merge given this is for demo purposes only.

### CODE REVIEW

File: examples/semantic_segmentation/semantic_segmentation/workflow.py [hunk 1]
```diff
@@ -37,12 +37,7 @@ class TestSample(Image):
 
 @dataclass(frozen=True)
 class GroundTruth(BaseGroundTruth):
```
Comment (sandalns) [inline examples/semantic_segmentation/semantic_segmentation/workflow.py:38]: If you pass BaseGroundTruth to define_workflow directly we can use `kolena.workflow.groundtruth` everywhere

Comment (nankolena) [reply examples/semantic_segmentation/semantic_segmentation/workflow.py:38]: I tried that initially, but the base `GroundTruth` class has to be sub-classed: https://github.com/kolenaIO/kolena/blob/2e3417d1e8e690aaafcb0c7b1f75eb51c9f92af4/kolena/workflow/ground_truth.py#L97

Comment (munkyshi) [reply examples/semantic_segmentation/semantic_segmentation/workflow.py:38]: Does this not work with `kolena.workflow.GroundTruth`?
```
>>> from kolena.workflow import GroundTruth
>>> issubclass(GroundTruth, GroundTruth)
True
```

**Comment by munkyshi** [comment]
Closing this out -- we've shipped a `no_op_evaluator`: https://docs.kolena.io/reference/workflow/evaluator/#kolena.workflow.evaluator_function.no_op_evaluator

**Pull request closed without merging**